pilot/pkg/security/authn/utils/utils.go

```diff
@@ -64,7 +65,9 @@ func BuildInboundTLS(mTLSMode model.MutualTLSMode, node *model.Proxy,
 		// protocol, e.g. HTTP/2.
 		ctx.CommonTlsContext.AlpnProtocols = util.ALPNHttp
 	}
-
+	if mc != nil && mc.MeshMTLS != nil && mc.MeshMTLS.CipherSuites != nil {
+		SupportedCiphers = mc.MeshMTLS.CipherSuites
```

Comment:
This is mutating a global variable. We cannot do that. Should set a local varaible to `ciphers := SupportedCiphers` and then override it in the If case

Reply:
Sure, took care.